fast-set.js

```diff
@@ -22,7 +22,9 @@ function FastSet(values, equals, hash, getDefault) {
     this.contentEquals = equals;
     this.contentHash = hash;
     this.getDefault = getDefault;
-    this.buckets = new this.Buckets(null, this.Bucket);
+    this.buckets = new this.Buckets(null, (function () {
+        return new this.Bucket();
+    }).bind(this));
```

Comment:
In V8, bind is slower than just capturing `this` as `self` in the parent scope and calling `new self.Bucket()`.